shell/shared/admin.js

```diff
@@ -759,6 +855,51 @@ if (Meteor.isServer) {
       }
 
       return { sent: true };
+    },
+    offerIpNetwork: function (token) {
+      if (!isAdmin()) {
+        if (tokenIsValid(token)) {
+          throw new Meteor.Error(403, "Offering IpNetwork is only allowed for logged in users " +
+            "(a token is not sufficient). Please sign in with an admin account")
+        } else {
+          throw new Meteor.Error(403, "User must be admin.");
+        }
+      }
+
+      var ipNetwork = new IpNetworkImpl(Meteor.userId());
+      var sturdyRef = waitPromise(ipNetwork.save({
+        sealFor: {
```

Comment:
You're right, it doesn't get passed through. The weird thing is that it actually works in this case, since it is ignored, and the union for ApiTokenOwner defaults to being a webkey anyways.